test/controllers/pregnancies_controller_test.rb

```diff
@@ -5,37 +5,37 @@ class PregnanciesControllerTest < ActionController::TestCase
     @user = create :user
     sign_in @user
     @patient = create :patient,
-                      name: 'Susie Everyteen', 
-                      primary_phone: '123-456-7890', 
+                      name: 'Susie Everyteen',
```

Comment:
I made a few edits to this in another PR on Monday -- it just looks like you linted it, so I'll merge this first and rebase mine.